capstone/backend/src/main/java/com/google/servlets/Item.java

```diff
@@ -45,9 +47,13 @@ public String getName() {
     return itemName;
   }
 
+  public String getBrand() {
```

Comment:
I also added itemSubtype, if you want to add functions for that too? Not necessary though

Reply:
I think I will just add it if we end up using it like just in case we never get to it